### PR TITLE
Development Branch

### PR DESCRIPTION
**Pull request by shakerAbuDrais**
We are done working on the app as a group and did the following: 

1- Divided the Tasks between the team members, using the kanban board.
2- Started to work with each student on his own tasks. 
3- Did the project Setup as a team
4- Fetched the Data from the APIs for (Rockets, Missions, and Dragons)
5- Displayed all the Rockets in the default route-page
6- Created Routes for other pages (Dragons & Missions) and Displayed the corresponding items in them.
7- Created functionality to reserve Rockets/Dragons and join a mission
8- Created myProfile page where a user can check his reserved Rockets/Dragons or missions he/she joined.
9- Added a test to see if each component is rendering
10- Tested the website and made sure that everything works as per requirements

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Ouch-gifs-38715704-365-199

You have submitted a project without a well documented README file. Kindly utilize the Microverse [readme template](https://github.com/microverseinc/readme-template) and include all the major headings. This will help users understand what your project is about.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,10 +1,31 @@
...
+## Built With
 
-## Available Scripts
+- HTML
+- CSS
+- JavaScript/React/Redux
 
-In the project directory, you can run:
+## Live Demo
+[Live Demo Link]()
```
Comment (Bondok6) [inline README.md:11]: - [OPTIONAL] The Live Demo link is broken, I think it would be better to fix or remove it for now 👍

File: README.md [hunk 2]
```diff
@@ -14,57 +35,43 @@ Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 The page will reload when you make changes.\
 You may also see any lint errors in the console.
 
-### `npm test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
```
Comment (Bondok6) [inline README.md:20]: - [OPTIONAL] I suggest to keep the `npm test` command because you already have some unit tests, so we need to use this command 💯

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi Team 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Great job using Redux ✅
- Well structured files ✅
- All tests are passed ✅
- No Linter errors ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] Great job adding some unit tests 💯 However, I can see that you have a space to add one more test for the profile component 👍

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**